lib/cobrato/resources/charge.rb

```diff
@@ -34,6 +34,12 @@ def cancel(id)
         end
       end
 
+      def destroy_or_cancel(id)
+        http.post("#{resource_base_path}/#{id}/destroy_or_cancel") do |response|
+          response.code == 204
```

Comment:
@jcomello e se a cobrança for cancelada e ao invés de excluída? Vai ser 204 o retorno?